packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/CSharpGen.cs

```diff
@@ -33,7 +34,7 @@ public async Task ExecuteAsync()
             Directory.CreateDirectory(Path.Combine(generatedSourceOutputPath, "Models"));
             List<Task> generateFilesTasks = new();
 
-            foreach (var model in output.Models)
+            foreach (TypeProvider model in output.Models) // now is of type TypeProvider and not ModelProvider so this doesn't compile
```

Comment:
Is this comment accurate?

Reply:
no - it was from a previous iteration. I will remove it